build-tools/scripts/jdk.mk

```diff
@@ -22,11 +22,15 @@
 #       It DOES NOT contain the -I itself; use $(JI_JDK_INCLUDE_PATHS:%=-I%) for that.
 #   $(JI_JVM_PATH):
 #       Location of the Java native library that contains e.g. JNI_CreateJavaVM().
+#   $(JI_JDK_BIN_PATH):
+#       Location of the JDK `/bin` directory, which contains `java/`javac`/etc.
 
 OS           ?= $(shell uname)
 JI_JAVAC_PATH = javac
 JI_JAR_PATH   = jar
 
+JI_JDK_BIN_PATH = $(shell which java)
```

Comment:
Shouldn't that be assigned to `$(dir $(shell which java))`?

Reply:
...and this is why we have code review...

Thanks!